### PR TITLE
Update OASys risk screens

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

This PR adds a non-OASys integration version of our placement considerations task, a tabbed sequence of 6 pages, each showing a column of risk widgets

Note that renaming sections is in a [later ticket](https://trello.com/c/pxHPmqse/1399-update-section-task-naming-and-ordering), meaning this PR does leave us in the awkward situation of having both a task and a section called "placement considerations"

#### Disability, cultural and specific needs
Disability, cultural and specific needs before

#### Requirements for placement (now food allergies)
food allergies before

#### Disability, cultural and specific needs
Disability, cultural and specific needs

#### Food allergies
food allergies

#### Task list
task list

#### Accommodation sharing
sharing
sharing-error-1
sharing error 2
sharing error 3

#### Cooperation
cooperation
cooperation error

#### Anti-social behaviour
anti social behaviour
ant social error 1
anti social error 2

#### Substance abuse
substance
substance error 1
substance error 2

#### RoSH level
rosh
rosh error

#### Risk management plan
risk man
risk man error

#### Check your answers
cya

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Update Oasys risk screens](https://trello.com/c/mepNDFWJ/1345-update-oasys-risk-screens)

File: server/views/applications/pages/placement-considerations/food-allergies/food-allergies.njk [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      },
+      {
+        value: "no",
+        text: "No"
+      }
+    ]
+  },fetchContext()) }}
+
+  {% set detailsHtml %}
+    <p>Temporary Accommodation placements provide a welcome pack that includes food.</p>
```
Comment (richpjames) [inline server/views/applications/pages/placement-considerations/food-allergies/food-allergies.njk]: I think we should have `class="govuk-body"` on all `<p>`s

File: server/views/applications/pages/assess-risks-for-placement/placement-considerations/accommodation-sharing.njk [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+{% set columnClasses = "govuk-grid-column-full" %}
+{% block questions %}
+
+  {% include "./partials/_navigation.njk" %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% set accommodationSharingYesDetailWarningHtml %}
+        <p>You will need to discuss the suitability of John Smith sharing with the community probation practitioners for the other occupants during placement.</p>
```
Comment (richpjames) [inline server/views/applications/pages/assess-risks-for-placement/placement-considerations/accommodation-sharing.njk]: _think_ we need the body class here too

**Review by richpjames — APPROVED** [review]
LGTM.

**Comment by selina-dxw** [comment]
TYSM for these. A few small changes:

If no information is provided when the user selects ‘Yes’ on anti-social behaviour, update the error message to:
'You must specify how you will support the person’s placement given the concerns or risks relating to anti-social behaviour'

Substance misuse:
One typo at the end of the error message, ‘issue’ should be ‘issues'

CYA second question:
'How will you support XX placement considering any risks to the support worker'

**Pull request merged**